### PR TITLE
omega-rpg: fix build on Linux

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by danielnachun** [comment]
I realized the blindingly obvious solution here was just to only make the symlink if it didn't exist.  This should be ready now.

File: Formula/omega-rpg.rb [hunk 1]
```diff
@@ -41,6 +43,7 @@ def install
 
   def post_install
     # omega refuses to run without license.txt in OMEGALIB
-    ln_s prefix/"license.txt", libexec/"license.txt"
+    license_file = libexec/"license.txt"
+    ln_s prefix/"license.txt", license_file unless license_file.exist?
```
Comment (carlocab) [inline Formula/omega-rpg.rb:47]:
```suggestion
    license_file = libexec/"license.txt"
    prefix.install_symlink license_file unless license_file.exist?
```

**Review by SMillerDev — APPROVED** [review]

File: Formula/omega-rpg.rb [hunk 2]
```diff
@@ -41,6 +43,7 @@ def install
 
   def post_install
     # omega refuses to run without license.txt in OMEGALIB
-    ln_s prefix/"license.txt", libexec/"license.txt"
+    license_file = libexec/"license.txt"
+    prefix.install_symlink license_file unless license_file.exist?
```
Comment (cho-m) [inline Formula/omega-rpg.rb:47]: Note that this doesn't look right. We are now creating a symlink to non-existent file as symlink is in opposite direction of original.

Should be fixed in separate PR to properly post-install for existing macOS bottles, maybe something like:
```suggestion
    libexec.install_symlink prefix/"license.txt" unless (libexec/"license.txt").exist?
```
